libs/pmp-web/user/single-user-statistics/feature/src/lib/containers/single-user-statistics/single-user-statistics.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="user-statistics">
+  <div *ngFor="let repo of mockedData">
+    <pmp-picture-label
```

Comment:
as i saw in the table component icon and its name is in separated columns in a table, maybe try to do this in the same way